### ISSUE TITLE
Pre-trained model

### ISSUE THREAD

**Issue by mrcabellom** [yeyuxmf/auto_tooth_arrangement#8]
Could you enable again the link of the pre-trained model?

**Comment by yeyuxmf** [comment]
@mrcabellom Sorry, I only saw a reply from you now. The existing model was mistakenly deleted on the online drive, and I don't have it either. As long as the environment is installed and the optimizer is changed to Adam, this code can be directly trained using the data I provide. If you feel that the final loss value is not small enough, increase the number of epochs, and the loss value will be further reduced.

**Comment by yeyuxmf** [comment]
If you want to train your data, you need to change the data preprocessing code and calculate your rotation and translation matrices.

**Comment by smoothumut** [comment]
Hi @huang229 thanks for the great work, the model's link looks broken again.
